quasar/main.tf

```diff
@@ -294,6 +294,78 @@ resource "aws_db_parameter_group" "quasar-qa" {
   #  }
 }
 
+resource "aws_db_parameter_group" "quasar-qa-pg11" {
+  name   = "quasar-qa-pg11"
+  family = "postgres11"
+
+  # Required for DMS CDC Replication:
+  # https://docs.aws.amazon.com/dms/latest/userguide/CHAP_Source.PostgreSQL.html#CHAP_Source.PostgreSQL.v10
+  parameter {
+    name         = "rds.logical_replication"
+    value        = "1"
+    apply_method = "pending-reboot"
+  }
+
+  # Required for DMS CDC Replication:
+  # https://docs.aws.amazon.com/dms/latest/userguide/CHAP_Source.PostgreSQL.html#CHAP_Source.PostgreSQL.v10
+  parameter {
+    name         = "max_logical_replication_workers"
+    value        = "500"
+    apply_method = "pending-reboot"
+  }
+
+  # Recommended by PGTuner tool: https://pgtune.leopard.in.ua/#/
```

Comment:
Just curious if you think we should document the assumptions we input in this tool to get the recommendations.

Reply:
We can add some more nuance here, but the assumption I think started out as "we don't know what the hell we're doing, this is a good starting point." I think we've noted that in the original setup ticket for this in Pivotal which I think is a better place for extended comments. We could possibly link to that ticket